src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
+@import './stars';
+@import './variables';
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto , sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white-color;
+  width: 200px;
+  height: $card-height;
+  flex-shrink: 0;
+  justify-content: center;
+  justify-items: center;
+
+  &__image {
+    margin: 32px 19px 28px;
+    width: 160px;
+    height: 134px;
+    flex-shrink: 0;
+  }
+
+  &__text {
+    max-width: 100%;
+    margin: 8px 16px 0;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    &-code {
+      margin: 4px 0 0 16px;
+      color: $secondary;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__review {
+    margin: 16px;
+    text-align: center;
+    justify-items: center;
+    justify-content: space-between;
+    display: flex;
+
+    &--stars :nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--score {
+      margin:0;
```

Comment:
you could reset the margin for a list of tags (h2, p, etc) as you did for body instead of resetting it for every element